jasmine/spec/inverted-index-test.js

```diff
@@ -0,0 +1,71 @@
+const books = require('./books.json')
+
+
+describe('InvertedIndex Class', () => {
+  beforeEach(() => {
```

Comment:
What's the implication of `beforeEach` as compared to `beforeAll`? Do you think you need to call these methods before each of the specs?